src/core/generator/generator.rs

```diff
@@ -28,8 +29,11 @@ pub struct Generator {
 pub enum Input {
     Json {
         url: Url,
+        method: Method,
+        body: serde_json::Value,
         response: Value,
         field_name: String,
+        operation_type: GraphQLOperationType,
```

Comment:
Storing `is_mutation` here will reduce the load on the caller.

Reply:
done